recipes/mg5amcnlo/meta.yaml

```diff
@@ -0,0 +1,161 @@
+{% set name = "mg5amcnlo" %}
+{% set version = "3.5.7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/hep-packaging-coordination/mg5amcnlo-release-mirror/releases/download/{{ version }}/MG5_aMC_v{{ version }}.tar.gz
+    sha256: ca3e027f078438318bd4143a102aebc2d51358f537b5d52bbc574fe49a6a901e
+    folder: source
+    patches:
+      # FIXME: mg5amcnlo assumes that ninja and collier are only static libraries
+      - allow-for-shared-libraries.patch
+
+build:
+  # FIXME: mg5amcnlo doesn't support Python 3.12 yet
+  skip: true  # [py<37 or py>311 or win]
```

Comment:
At the moment this could technically be `noarch`, but as the goal is to add compiled components to support Pythia8 in the future, it would have to move away from `noarch`, so might as well start off as non-`noarch`.